kuma/wiki/tests/conftest.py

```diff
@@ -175,12 +218,56 @@ def doc_hierarchy(wiki_user):
         title='Bottom Document',
         created=datetime(2017, 4, 24, 13, 52)
     )
-
+    DocumentZone.objects.create(
+        document=top_doc,
+        css_slug='lindsey',
+        url_root='fleetwood-mac'
+    )
+    DocumentZone.objects.create(
+        document=middle_top_doc,
+        css_slug='bobby',
+        url_root='spinners'
```

Comment:
😃 Bobby Smith was one of the lead singers for the [Spinners](https://en.wikipedia.org/wiki/The_Spinners_(American_R%26B_group)).